### PR TITLE
LIVE-601 - Market page analytics

### PR DESCRIPTION
**Pull request by nparigi-ledger**
Adding tracking to multiple market page actions.

New events :
Page Market
Page Market Search 
Page Market Query
Page Market Favourites
Page Market Coin
Buy Crypto Page Market Coin
Swap Crypto Page Market Coin

### Type

Feature

### Context

[LIVE-601]
Specs : https://ledgerhq.atlassian.net/wiki/spaces/WALLETCO/pages/2626453631/MARKET+-+Live+specifications

### Parts of the app affected / Test plan

Market page, Market page coin

edit 03/02: there is some change to be done following feedback from product, PR is paused

[LIVE-601]: https://ledgerhq.atlassian.net/browse/LIVE-601?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by juan-cortes** [comment]
Please fill the template for the PR correctly 🙏🏼

File: src/screens/Market/index.tsx [hunk 1]
```diff
@@ -74,16 +105,18 @@ const BottomSection = ({
     [t],
   );
 
-  const onChange = useCallback(
-    (value: any) => {
-      refresh(value);
-    },
-    [refresh],
-  );
+  const onChange = (value: any) => {
```
Comment (ofreyssinet-ledger) [inline src/screens/Market/index.tsx:108]: why the change to an arrow function ? this will cause useless rerenders
